OrbitBase/Platform/Windows/Profiling.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// clang-format off
+#include <OrbitBase/Platform/Windows/Profiling.h>
+// clang-format on
+
+#include <processthreadsapi.h>
+
+#include <OrbitBase/Logging.h>
+#include <string>
+
+template <typename T>
+inline T GetProcAddress(const std::string& library, const std::string& procedure) {
```

Comment:
I do not think inline does anything here. in any case I would leave it to compiler... also maybe make it static.

Reply:
Actually this should be in a header. It could be reused elsewhere in the code base.